test/test_file_converter.py

```diff
@@ -32,19 +34,28 @@ def test_table_removal(Converter, xpdf_fixture):
     # assert text is retained from the document.
     # As whitespace can differ (\n," ", etc.), we standardize all to simple whitespace
     page_standard_whitespace = " ".join(pages[0].split())
-    assert "Adobe Systems made the PDF specification available free of charge in 1993." in page_standard_whitespace
+    assert (
+        "Adobe Systems made the PDF specification available free of charge in 1993."
+        in page_standard_whitespace
+    )
```

Comment:
How about we move this assert to the `test_convert()` use case? Then we can run it also with PDFToTextOCRConverter and do not mix it with the table removal test case.